packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -943,30 +970,47 @@ export default decorate([
                         </FormGroup>
                       )}
                       {state.isFull && (
+                        <div>
```

Comment:
you can delete this `<div>` and its closure.

Reply:
It isn't possible due to the condition.